openduck-py/openduck_py/routers/voice.py

```diff
@@ -350,6 +366,7 @@ def _check_for_exceptions(response_task: Optional[asyncio.Task]) -> bool:
             print("response task was cancelled")
         except Exception as e:
             print("response task raised an exception:", e)
+            print(traceback.format_exc(e))
```

Comment:
nice

Reply:
except for the bit where it raises an exception of its own somehow :p